macros/parsers/parserMultiAnswer.pl

```diff
@@ -488,6 +518,13 @@ =head2 namedRules
 if you need to intersperse other rules with the ones for the C<MultiAnswer>. In this case, you must
 use C<NAMED_ANS> instead of C<ANS>. Default: 0.
 
+=head2 cmpOpts
+
+This is a hash of options that will be passed to the cmp method of all answers. For example,
+C<< cmpOpts => { weight => 0.5 } >>. This option is provided to make it more convenient to pass
+options to all answers when utilizing PGML. To pass options to a single answer, use the
+C<< setCmpFlags($which_rule, %flags) >> method instead. Default: undef (no options are sent).
```

Comment:
This documentation is not correct.  Regardless of if `singleResult` is set, this is not a substitute for the `setCmpFlags` method, and in no case are these flags actually passed to the `cmp` method for all answers.  There are two sets of answer evaluators that a MultiAnswer object uses.  There is a set of answer evaluators that are associated with the answer rules (there is only one of these if `singleResult` is set, and one for each answer otherwise), and there are answer evaluators for each answer that are called internally by the `entry_check` or `single_check` methods (there are always as many of these as there are answers regardless of the `singleResult` setting).  The only way to set flags for the internally called answer evaluators you must use the `setCmpFlags` method.  The things set in `cmpOpts` won't be passed to those evaluators.

Reply:
Thanks for the explanation. I got confused when I thought it was implied at the meeting I could have used that method to add cmp options. I have updated the docs to mirror what you have in parserRadioMultiAnswer.